docs/reference/sql/endpoints/client-apps/tableau-desktop.asciidoc

```diff
@@ -0,0 +1,54 @@
+[role="xpack"]
+[testenv="platinum"]
+[[sql-client-apps-tableau-desktop]]
+=== Tableau Desktop
+
+Use the {es} JDBC driver and dedicated {es} Tableau Connector to access {es} data from Tableau Desktop.
+
+IMPORTANT: Elastic does not endorse, promote or provide support for this application; for native Elasticsearch integration in this
+product, please reach out to its vendor.
+
+==== Prerequisites
+
+* https://www.tableau.com/products/desktop[Tableau Desktop] 2019.4 or later
+* {es-sql} <<sql-jdbc, JDBC driver>>
+* https://www.elastic.co/downloads/tableau-connector[{es} Connector for Tableau]
+
+==== Data loading
+
+First place the downloaded JDBC driver in Tableau Desktop drivers directory:
+
+* Windows: `C:\Program Files\Tableau\Drivers`
+* Mac: `/Users/[user]/Library/Tableau/Drivers`
+
+and the {es} Connector for Tableau in Tableau Desktop connectors directory:
+
+* Windows: `C:\Users\[Windows User]\Documents\My Tableau Repository\Connectors`
+* Mac: `/Users/[user]/Documents/My Tableau Repository/Connectors`
```

Comment:
This inconsistent way of specifying the username has somehow trickled down from Tableau's docs - see `How to use a connector plugin` section on [Tableau's page](https://help.tableau.com/current/pro/desktop/en-us/examples_connector_sdk.htm). I'm not a fan, but it seems to be consistent with [other pages](extensiongallery.tableau.com/connectors/179) for some reason.